Packages/com.unity.multiplayer.samples.coop/Utilities/EditorChildSceneLoader.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+/// <summary>
+/// Should be placed inside root scene and setup with child scenes to load. ChildSceneLoader will take care of the rest
+/// </summary>
+public class EditorChildSceneLoader : MonoBehaviour
+{
+    [SerializeField]
+    public List<SceneAsset> ChildScenesToLoadConfig;
```

Comment:
Maybe turn this Config list into a reference to a new type `CompositeSceneConfig : ScriptableObject` which would contain the list itself (and whatever other parameters we might want?).

Reply:
Yeah I considered using a SO for this, but I felt associating this with the scene directly made more sense? The root scene would manage it's own list of child scenes, the association would be more embedded? From a UX perspective that seemed to make more sense, any reason we'd want to go with SOs?